Sources/MediaCategoryCells/MediaCollectionViewCell.swift

```diff
@@ -57,21 +57,21 @@ class MediaCollectionViewCell: BaseCollectionViewCell {
         thumbnailView.layer.masksToBounds = true
         thumbnailView.layer.cornerRadius = thumbnailView.frame.size.width / 2.0
         titleLabel.text = audiotrack.title
-        descriptionLabel.text = audiotrack.albumTrack?.artist?.name ?? NSLocalizedString("UNKNOWN_ARTIST", comment: "")
+        descriptionLabel.text = name(of: audiotrack.albumTrack?.artist)
         newLabel.isHidden = !audiotrack.isNew
         thumbnailView.image = audiotrack.thumbnailImage()
     }
 
     func update(album: VLCMLAlbum) {
-        titleLabel.text = album.title != "" ? album.title : NSLocalizedString("UNKNOWN_TITLE", comment: "")
-        descriptionLabel.text = album.albumArtist?.name != "" ? album.albumArtist?.name : NSLocalizedString("UNKNOWN_ARTIST", comment: "")
+        titleLabel.text = album.isUnknownAlbum() ? NSLocalizedString("UNKNOWN_ALBUM", comment: "") : album.title
```

Comment:
I would hide that behind a view model function/ computed property that we use everywhere  instead of having these checks everywhere